lua/tardis/parts/door.lua

```diff
@@ -113,11 +113,19 @@ else
             self.DoorPos = self.exterior.DoorOverride or
                 math.Approach(self.DoorPos, self.DoorTarget, FrameTime() * (1 / animtime))
 
+            -- for extension tweaks
```

Comment:
there's one scenario: you use the locked the TARDIS, and the door wiggles a tiny bit without opening

to achieve that, I change its position slightly and it finishes the close animation 

I added this code so that the interior door wouldn't do that